x-pack/plugins/security_solution/public/common/components/ml/tables/anomalies_user_table.tsx

```diff
@@ -65,7 +65,7 @@ const AnomaliesUserTableComponent: React.FC<AnomaliesUserTableProps> = ({
     return null;
   } else {
     return (
-      <Panel loading={loading}>
+      <Panel loading={loading} data-test-subj="user-anamolies-tab">
```

Comment:
spelling, not sure where this is used either

Reply:
Thank for catching it. It is used by a cypress test.